hexrd/ui/interactive_template.py

```diff
@@ -27,11 +23,48 @@ def __init__(self, parent=None):
         self.translation = [0, 0]
         self.complete = False
         self.event_key = None
-        self.parent.setFocusPolicy(Qt.ClickFocus)
+        self.detector = detector
+        self.instrument = instrument
+        self._static = True
+
+        for canvas in self.image_tab_widget.active_canvases:
+            canvas.setFocusPolicy(Qt.ClickFocus)
+
+    @property
+    def current_canvas(self):
+        idx = self.image_tab_widget.current_index
+        return self.image_tab_widget.active_canvases[idx]
+
+    @property
+    def ax(self):
+        idx = HexrdConfig().current_imageseries_idx
+        return self.current_canvas.axes_images[idx]
 
     @property
     def raw_axes(self):
-        return list(self.parent.raw_axes.values())[0]
+        if not self.current_canvas.raw_axes:
+            return self.current_canvas.axis
```

Comment:
Maybe the property `raw_axes` should just be renamed to `axis` since it is not necessarily raw?

Reply:
That makes sense, updated!